### PR TITLE
Upgrade accessibility.cloud widget to v3.2.1 (Part II)

### PR DESCRIPTION
**Pull request by opyh**
- Fix the issue we had with the JS minifier when deploying the changes in #604
- Add 2 missing translation strings
- CSS fix for bottom padding below places nearby results

### CODE REVIEW

File: app/views/nodes/_node_nearby.html.haml [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
+        limit: 10,
+        excludeSourceIds: ['LiBTS67TjmBcXdEmX'], // excludes Wheelmap's own PoIs from results
+      },
+      onSuccess: function() {
+        containerElement.classList.add('has-results');
+      }
+    });
+  }
+
+%script{type:"text/javascript", async: true, defer: true, src: "https://cdn.jsdelivr.net/gh/sozialhelden/accessibility-cloud-js@3.2.1/build/example/accessibility.cloud-3.2.1-21a6710d.min.js"}
```
Comment (Hoverbear) [inline app/views/nodes/_node_nearby.html.haml]: Might be good to use https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

Comment (opyh) [inline app/views/nodes/_node_nearby.html.haml]: Done!

**Comment by holgerd** [comment]
What is the next step here? @Hoverbear to review this, correct?

**Comment by Hoverbear** [comment]
@holgerd Yes :)

**Comment by 1000miles** [comment]
@holgerd cc @opyh 

Deployed to staging with release [20170530082716](https://github.com/sozialhelden/wheelmap/releases/tag/staging-20170530082716) today. 

Example node: https://staging.wheelmap.org/nodes/2708872307

### Screens: 

#### Firefox (disabled privacy blocker)



#### Chrome (disabled privacy blocker)

**Comment by 1000miles** [comment]
@opyh cc @holgerd @anngreenberg 

#### Adding the screenshot with enabled privacy blocker

screen shot 2017-05-31 at 15 09 19

**Comment by opyh** [comment]
Hmm. I think privacy badger has a point there. It would be better to not rely on jsdelivr, but to serve the script through our own CDN. I'm about to set up fastly for accessibility.cloud soon anyway, then we can move it there. I'd keep the issue open until then. What do you think?

**Comment by 1000miles** [comment]
@Hoverbear Thanks but the concern was rather how do I know of the AC widget existence if I permanently have my privacy blocker enabled and there is no hint about it to turn it on in order to see it. :)

@opyh Sounds good to less rely on external CDNs. Let's continue the discussion directly on the Github issue #623 to keep overview?

**Pull request merged**